src/sentry/api/event_search.py

```diff
@@ -343,6 +347,15 @@ def visit_paren_term(self, node, children):
 
         return self.flatten(children[1])
 
+    def visit_boolean_filter(self, node, children):
+        (search_key, _, search_value) = children
+
+        if search_key.name in self.boolean_keys:
+            search_value = SearchValue({"true": 1, "false": 0}.get(search_value.text.lower()))
+            return SearchFilter(search_key, "=", search_value)
```

Comment:
Do you want to handle negations as well? Someone could foolishly do `event.handled!:true`

Reply:
👀 Dang I totally can see someone doing that... I'll add that as well.